extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -506,6 +506,7 @@ def to_dict(self):
         model['_c_dl'] = copy.deepcopy(self._c_dl)
         model['_c_lw'] = copy.deepcopy(self._c_lw)
         model['_c_l'] = copy.deepcopy(self._c_l)
+        model = self._make_json_serializable(model)
         return model
 
     def from_dict(self, model):
```

Comment:
If this PR gets merged within two days, then yes. Otherwise I recommend making the change in this PR or a separate one so that we don't forget about it.